documentation/projects/proposals/trust_and_safety/content_report_moderation/20230411-project_proposal_content_report_moderation.md

```diff
@@ -0,0 +1,338 @@
+# 2023-04-11 Project Proposal
+
+## Reviewers
+
+<!-- Choose two people at your discretion who make sense to review this based on their existing expertise. Check in to make sure folks aren't currently reviewing more than one other proposal or RFC. -->
+
+- [ ] @zackkrida
+- [ ] @olgabulat
+
+## Project summary
+
+<!-- A brief one or two sentence summary of the project's features -->
+
+While Openverse supports reporting results as mature, we do not have a process
+of handling these reports. Likewise, there are significant gaps in our
+infrastructure that make it difficult to respond to these reports or to lean on
+contributors outside the core maintainers for help processing them. This project
+will address these gaps, make it possible to easily and safely respond to
+content reports, and develop a baseline process for moderating content based on
+reports. It will also rename the "mature content" reports to "sensitive content"
+reports. This project is not the end-all, be-all for content report moderation
```

Comment:
That's a great idea. I'd actually like to create a separate document for that, as it'd undoubtedly be referenced by other projects in the future as well. Maybe a "Terms" page at the root of the Trust & Safety directory next to the initial overview & user stories would be a good place to hold ideas like that. Or maybe more generally "guiding principles"? There are some aspects of the sensitive terms IP that we could add there too, like the "this will never be perfect" bit you wrote, which I think is helpful to keep in mind as we work through these things.

Reply:
A "guiding principles" page sounds like a great idea. I'll resolve this concern.